tardis/configuration/utilities.py

```diff
@@ -1,17 +1,19 @@
+from cobald.daemon.plugins import YAMLTagSettings
 import yaml
 
 
 def enable_yaml_load(tag):
     def yaml_load_decorator(cls):
         def class_factory(loader, node):
+            settings = YAMLTagSettings.fetch(cls)
             new_cls = cls
             if isinstance(node, yaml.nodes.MappingNode):
-                parameters = loader.construct_mapping(node)
+                parameters = loader.construct_mapping(node, deep=settings.eager)
```

Comment:
This is necessary since MFA is using nested lists and dictionaries, when initialised using a yaml tag. This function is only used in the unittest. In real life it will be evaluated by COBalD, which already supports eager evaluation of yaml tags.